{{ cookiecutter.project_slug }}/requirements.txt

```diff
@@ -1,4 +1,4 @@
-jina[devel]
+jina[devel]==0.4.1
```

Comment:
```suggestion
jina==0.5.0
```

Reply:
bumped version here and updated ```cookiecutter.json``` version to ```0.0.3``` (confirming if this version is right)